### PR TITLE
test: WebdriverIO CI

### PR DESCRIPTION
**Pull request by christian-bromann**
Similar to https://github.com/GoogleChromeLabs/chromium-bidi/pull/1531 this patch adds WebdriverIO integration tests.

### CODE REVIEW

File: .github/workflows/webdriverio.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  pull_request:
+  push:
+    branches: 'main'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  selenium:
```
Comment (sadym-chromium) [inline .github/workflows/webdriverio.yml:19]:
```suggestion
  webdriver-io:
```

File: .github/workflows/webdriverio.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        id: browser
+        run: node tools/install-browser.mjs --github
+      - name: Install chromedriver for the pinned browser
+        id: driver
+        run: node tools/install-browser.mjs --chromedriver --github
+      - name: Run WebdriverIO test
+        timeout-minutes: 20
+        run: >
+          xvfb-run --auto-servernum
+          node tests_external/selenium.mjs
```
Comment (sadym-chromium) [inline .github/workflows/webdriverio.yml:51]:
```suggestion
          node tests_external/webdriverio.mjs
```

File: tests_external/helpers/index.mjs [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import {execSync} from 'child_process';
```
Comment (sadym-chromium) [inline tests_external/helpers/index.mjs:1]: It perfectly make sense to move this logic to a common place, but I think it's better to move it to `tools`: https://github.com/GoogleChromeLabs/chromium-bidi/pull/1616

**Review by sadym-chromium — COMMENTED** [review]
Nice job, thanks! Waiting for the https://github.com/GoogleChromeLabs/chromium-bidi/pull/1616 to be merged.

File: package.json [hunk 4]
```diff
@@ -229,6 +229,7 @@
     "terser": "5.25.0",
     "tslib": "2.6.2",
     "typescript": "5.3.2",
+    "webdriverio": "^8.26.1",
```
Comment (Lightning00Blade) [inline package.json:232]: Please pin the version. Dependabot will handle updating it.

Comment (sadym-chromium) [inline package.json:232]:
```suggestion
    "webdriverio": "8.26.1",
```

Comment (Lightning00Blade) [reply package.json]: We can't use suggestion to update this as it does not update the `package-lock.json`.

**Review by sadym-chromium — APPROVED** [review]

**Comment by OrKoN** [comment]
FYI we plan to remove this test in https://github.com/GoogleChromeLabs/chromium-bidi/pull/4156 to reduce the number of dependencies. I trust that you have sufficient test coverage downstream and I do not expect this test to catch things that WPT do not catch.

**Pull request merged**